BZ2187603: SCC and SA for DPDK checkup (#58871)
openshift/openshift-docs: modules/virt-measuring-latency-vm-secondary-network.adoc

Reviewer: Why is this file part of this PR?
The PR is supposed to address DPDK checkup documentation, while this file is about the latency checkup.

Author: This PR includes some general doc housekeeping.

Reviewer: Ack

:_content-type: PROCEDURE
[id="virt-measuring-latency-vm-secondary-network_{context}"]
= Virtual machine latency checkup

You use a predefined checkup to verify network connectivity and measure latency between two virtual machines (VMs) that are attached to a secondary network interface. The latency checkup uses the ping utility.

You run a latency checkup by performing the following steps:

. Create a service account, role, and role bindings to provide cluster access permissions to the latency checkup.

> . Create a config map. The config map provides the input to run the checkup and stores the results.

Reviewer 2: s/Create a config map. The config map provides/Create a config map to provide (minimalism)

> . Create a config map. The config map provides the input for the framework to run the checkup and stores the results.

Reviewer 3: ```suggestion
. Create a config map. The config map provides the input for the checkup and also stores the results.
```

. Create a job to run the checkup.
. Review the results in the config map.

> . Optional: To rerun the checkup, delete the existing job and config map and then create a new job and config map.

Reviewer 3: ```suggestion
. Optional: To rerun the checkup, delete the existing job and config map and then create a new config map and job.
```
